tests/test_units.py

```diff
@@ -1,12 +1,158 @@
-import mammos_units as mu
+import mammos_units as u
+import pytest
+import numpy as np
+from astropy.units import isclose
+
 
 def test_new_units():
     # Formula Unit
-    fu = mu.f_u_
-    assert fu.si == mu.dimensionless_unscaled
-    assert fu == mu.Formula_Unit
-    
-    # Bohr magneton
-    mu_B = mu.mu_B
-    assert mu_B.si == mu.constants.muB
-    
\ No newline at end of file
+    assert hasattr(u, "f_u")
+    assert u.f_u == u.formula_unit
+
+    # Bohr Magneton
+    assert hasattr(u, "mu_B")
+    assert u.mu_B == u.constants.muB
+
+    # Atom
+    assert hasattr(u, "atom")
+
+
+def test_equivalency_creation():
+    """Test creating equivalency objects with different volumes"""
+    vol1 = 100 * u.angstrom**3
+
+    eq1 = u.moment_induction(vol1)
+
+    assert eq1 is not None
+    assert eq1.name == ["moment_induction"]
+
+    # Test with different volume units
+    vol2 = 1e-28 * u.m**3
+    eq2 = u.moment_induction(vol2)
+    assert eq2 is not None
+
+
+@pytest.mark.parametrize("counting_unit", [u.f_u, u.atom])
+def test_moment_induction_equivalency(counting_unit):
+    """Test conversion between mu_B/f_u and Tesla"""
+    vol = 100 * u.angstrom**3
+    vol_m3 = vol.to(u.m**3)
+    eq = u.moment_induction(vol)
+
+    # Test forward conversion: mu_B/counting_unit → Tesla
+    moment = 2.5 * u.mu_B / counting_unit
+    polarisation = moment.to(u.T, equivalencies=eq)
+    expected = 2.5 * u.constants.muB * u.constants.mu0 / vol_m3
+    assert abs(polarisation.value - expected.value) < 1e-12
```

Comment:
An alternative could be to use `np.isclose`? But it is purely a preference.

Reply:
Good plan!